nix/formatters.nix

```diff
@@ -1,36 +0,0 @@
-# This file is part of the IOGX template and is documented at the link below:
```

Comment:
We run this command as a formatting step:

```
purs-tidy format-in-place 'src/**/*.purs'
```

@zeme-wana do you think we can add check for it?

Reply:
if inside `nix develop`, this now becomes `pre-commit run purs-tidy`